intrinsic_cal/CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ set(CMAKE_CXX_FLAGS ${CMAKE_CSS_FLAGS} "-fPIC")
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-  INCLUDE_DIRS include /usr/include/eigen3
+  INCLUDE_DIRS /usr/include/eigen3
```

Comment:
I've mentioned that [here](https://github.com/ros-industrial/industrial_calibration/pull/56#discussion_r43195467).
`FIND_PACKAGE(Eigen3 REQUIRED)` works fine, even `FIND_PACKAGE(Eigen REQUIRED)` will work because Ceres installs such a file. See https://github.com/ros-industrial/industrial_calibration/pull/60